ui/src/components/QMediaPlayer.js

```diff
@@ -134,6 +134,10 @@ export default {
     radius: {
       type: [Number, String],
       default: 0
+    },
+    seekable: {
+      type: Boolean,
+      default: true
```

Comment:
Hi, sorry I missed this PR. I prefer some changes.
1. props should never be true by default. Change the prop to 'noSeek'
2. Add API information to QMediaPlayer.json
3. add example to demo (docs)
Thanks! :heart: